Formula/yelp-tools.rb

```diff
@@ -12,20 +12,45 @@ class YelpTools < Formula
     sha256 "66721baa2d6c6978cceac29f35e8112e96804b6ae0197d89175867f17caca2cd" => :mavericks
   end
 
+  depends_on "gettext" => :build
+  depends_on "gtk+3" => :run
+  depends_on "intltool" => :build
   depends_on "itstool" => :build
-  depends_on "pkg-config" => :build
-  depends_on "libxslt" => :build
   depends_on "libxml2" => :build
-  depends_on "yelp-xsl"
+  depends_on "libxslt" => :build
+  depends_on "pkg-config" => :build
+
+  resource "yelp-xsl" do
+    url "https://download.gnome.org/sources/yelp-xsl/3.18/yelp-xsl-3.18.0.tar.xz"
+    sha256 "893620857b72b3b43ee3b462281240b7ca4d80292f469552827f0597bf60d2b2"
+  end
 
   def install
+    resource("yelp-xsl").stage do
+      system "./configure", "--disable-debug",
+                            "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{prefix}"
+      system "make", "install"
+      ENV.append_path "PKG_CONFIG_PATH", "#{prefix}/share/pkgconfig"
```

Comment:
This syntax works, I tried this but it had no effect (ENV was the same after):
```
      ENV.append_path "PKG_CONFIG_PATH", "#{lib}/pkgconfig"
```
However, the syntax I'm using here doesn't match how other formulas do it, and "brew audit" complains:
```
yelp-tools:
  * "#{prefix}/share" should be "#{share}"
Error: 1 problem in 1 formula
```